### ISSUE TITLE
Display the link as was in the text

### ISSUE THREAD

**Issue by highfeed** [gregjacobs/Autolinker.js#127]
It is necessary to display the link as it was in the text

```
github.com - <a href="http://github.com">github.com</a>
https://google.com - <a href="https://google.com">https://google.com</a>
```

Example:
github.com - <a href="http://github.com">github.com</a>
https://google.com - <a href="https://google.com">https://google.com</a>

You do? Please new options.

**Comment by gregjacobs** [comment]
Hey, thinking of supporting this right out of the box. Everyone seems to want this :)

**Comment by gregjacobs** [comment]
Fixed in v0.21.0
